#!/Users/n01545735/MLHCBugs/conda/env/dipy.1.4.0/bin/python3.8
import sys
from lib2to3.main import main

sys.exit(main("lib2to3.fixes"))
